mailpoet/tests/integration/Segments/SegmentSubscribersRepositoryTest.php

```diff
@@ -247,10 +247,6 @@ private function createDynamicSegmentEntity(string $role = 'editor'): SegmentEnt
   }
 
   private function cleanup() {
-    $this->truncateEntity(SegmentEntity::class);
-    $this->truncateEntity(SubscriberEntity::class);
-    $this->truncateEntity(SubscriberSegmentEntity::class);
-    $this->truncateEntity(DynamicSegmentFilterEntity::class);
     $this->clearSubscribersCountCache();
```

Comment:
Where is the cache stored, is it a WP transient? Just thinking if we should add transients to our list of additional things to clean up. On the other hand, they may also stay in memory 🤔

Reply:
Ahh, we call that already in `_bootstrap` so removed: https://github.com/mailpoet/mailpoet/pull/4818/commits/557577a432821dcc8238308bb4ff7b349b3907a4